src/components/Slider/Slider.tsx

```diff
@@ -5,37 +5,36 @@ import { Dispatch, ReactElement, SetStateAction } from "react";
 import css from "./Slider.module.css";
 
 export interface IconItem {
+    id: string;
     icon: ReactElement<IconProps>;
+}
+
+export interface TextItem {
+    id: string;
     name: string;
 }
 
 export interface SliderProps {
-    items: string[] | IconItem[];
-    activeItem: string;
+    items: TextItem[] | IconItem[];
+    activeItemId: string;
     onChange: Dispatch<SetStateAction<string>>;
 }
 
-const isIconItem = (item: string | IconItem): item is IconItem => (item as IconItem).icon !== undefined;
+const isIconItem = (item: TextItem | IconItem): item is IconItem => (item as IconItem).icon !== undefined;
+
+export default function Slider({ items, activeItemId, onChange }: SliderProps): ReactElement<SliderProps> {
+    const getItemClasses = (id: string) => [css.item, id === activeItemId ? css.active : ""].join(" ");
 
-export default function Slider({ items, activeItem: active, onChange }: SliderProps): ReactElement<SliderProps> {
     return (
         <div data-test-id="Slider" className={css.wrapper}>
```

Comment:
actually since it is a "list" of items you could use the "semantically correct"  html element `<ul>` and items as `<li>` wdyt?